index.html

```diff
@@ -37,8 +38,200 @@ <h1>I'm Joseph Glad to meet you!</h1>
+            <p>
+                A daily selection of privately personalized reads; 
+                no accounts or sign-ups required.
+            </p>
+        </div>
+        
+         <div class="link">
+            <a href="">Html</a>
+            <a href="">css</a>
+            <a href="">javascript</a>
+         </div>
+
+         <button class="button"><a href="">See Project</a></button>
+        
+    </section>
+    <section id="secondPage">
+        <img class="snap" src="Images/Snapshoot Portfolio (3).png" alt="">
+        <h2 class="multi">Multi-Post
+            Stories</h2>
+        <div class="container">
+            <div class="canopy">
+                <p>CANOPY</p>
+            </div>
+            
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>Backend Dev</p>
+            </div>
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>2015</p>
+            </div>
+        </div>
+        <div class="content">
+            <p>
+                A daily selection of privately personalized reads; 
+                no accounts or sign-ups required.
+            </p>
+        </div>
+        
+         <div class="link">
+            <a href="">Html</a>
+            <a href="">css</a>
+            <a href="">javascript</a>
+         </div>
 
+         <button class="button"><a href="">See Project</a></button>
+        
+    </section>
+    <section class="about">
+        <h2 class="multi">About Myself</h2> 
+    <p>
+        I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.
+    </p>
+        <p class="contain">LET’S CONNECT</p>
     
+    <ul class="about-social"> 
+        <li> <a href=""><img src="Images/github.svg" alt=""></a></li>
+        <li> <a href=""><img src="Images/Linkedin icon.svg" alt=""></a></li>
+        <li> <a href=""><img src="Images/Vector.svg" alt=""></a></li>
+        <li> <a href=""><img src="Images/twitter.svg" alt=""></a></li>
+        <li> <a href=""><img src="Images/m vector.svg" alt=""></a></li>
+        
+        
+    </ul>
+
+    <button class="about-button"><a href="">Get my resume</a></button>
+</section>
+
+<div class="lang">
+    <h3>Languages</h3>
+    <img src="Images/arrow-down-sign-to-navigate.png" alt="">
+</div>
+
+<ul class=" basic"> 
+    <li><img src="Images/Ellipse 1.png" alt=""> <p>Javascript</p> </li>
+    <li><img src="Images/Ellipse 1 (1).png" alt=""> <p>HTML</p> </li>
+    <li><img src="Images/Ellipse 1 (2).png" alt=""> <p>CSS</p></li>
+</ul>
+
+<div class="lang-skills">
+    <h3>Frameworks</h3>
+    <img src="Images/right-arrow.png" alt="">
+</div>
+<div><img class="shape" src="Images/Separator Bottom.png" alt=""></div>
+<div class="lang-skills">
```

Comment:
- [ ] Kindly consider restructuring your `skills` with following order instead of taking divs for language, frameworks and skills, in order to meet the project requirement:
```
<ul>
  <li>Languages
    <ul>
      <li>HTML</li>
      <li>CSS</li>
      <li>Javascript</li>
    </ul>
  </li>
  <li>Frameworks</li>
  <li>Skills</li>
</ul>

```

Reply:
changes have been made as requested.